scripts/lake_sheltering_nlcd.R

```diff
@@ -1,4 +1,45 @@
 
+
+get_nlcd_classes <- function(lake.buffer, filename){
```

Comment:
yeah, this is an entirely new function and the one we use for modeling is unchanged